### ISSUE TITLE
Add ability to do multiple gleanerio and tenant configs

### ISSUE THREAD

**Issue by valentinedwv** [earthcube/scheduler#108]
This will be dependent upon #106 

It would be nice to have a fully centralized service when multiple config files could be

Option 1 is to run multiple workflows. Need to ensure that a project variable gets sent all the way through, and that a  ingest code container could be configured with some parameters that are different for each workflow
think this might help: https://docs.dagster.io/_apidocs/cli#dagster-api-grpc
or bit of a config pain, but each community might have an compose-override with a variable to namespace it.
see https://docs.docker.com/compose/compose-file/10-fragments/#example-4

Long term option we have a higher level config that just manages everything

**Comment by valentinedwv** [comment]
This is done by having a separate ingest and task stack for each 'configuration'
